modules/codemod/lib/v5/renameCoreImports.ts

```diff
@@ -0,0 +1,40 @@
+import {API, FileInfo} from 'jscodeshift';
+
+type SourceMap = {
+  '@workday/canvas-kit-react/core': '@workday/canvas-kit-react/tokens';
+  '@workday/canvas-kit-labs-react/core': '@workday/canvas-kit-labs-react/tokens';
+};
+
+const sourceMap: SourceMap = {
+  '@workday/canvas-kit-react/core': '@workday/canvas-kit-react/tokens',
+  '@workday/canvas-kit-labs-react/core': '@workday/canvas-kit-labs-react/tokens',
+};
```

Comment:
What's the need to type this twice? Why not:

```suggestion

const sourceMap = {
  '@workday/canvas-kit-react/core': '@workday/canvas-kit-react/tokens',
  '@workday/canvas-kit-labs-react/core': '@workday/canvas-kit-labs-react/tokens',
} as const;
```

 Below you'd have to also change `as keyof SourceMap` to `as keyof typeof sourceMap`, but personally I prefer that. It reduces the Typescript noise.

Reply:
Yeah, I could just do the one. 👍🏻